default.nix

```diff
@@ -26,46 +30,56 @@ let
     noTest = pkg: haskell.lib.dontCheck pkg;
 
     hspkgs = haskell.packages.${bootghc};
+    ghc    = haskell.compiler.${bootghc};
 
     ourtexlive =
       nixpkgs.texlive.combine {
         inherit (nixpkgs.texlive) scheme-small collection-xetex fncychap titlesec tabulary varwidth framed capt-of wrapfig needspace dejavu-otf helvetic; };
     fonts = nixpkgs.makeFontsConf { fontDirectories = [ nixpkgs.dejavu_fonts ]; };
     docsPackages = if withDocs then [ python3Packages.sphinx ourtexlive ] else [];
 
-    deps =
-      [ autoconf automake m4
+    depsSystem = with stdenv.lib; (
+      [ arcanist
```

Comment:
arcanist left that list, just a few hours ago =)

Reply:
Done!